src/pytest_plugins/filler/tests/test_filler.py

```diff
@@ -102,9 +102,57 @@ def test_shanghai_two(state_test, x):
                 Path("fixtures/state_tests/shanghai/module_shanghai/shanghai_two.json"),
             ],
             [2, 2, 2, 2, 2, 2, 2, 2, 2, 6, 6, 6],
-            False,
+            True,
             id="default-args",
         ),
+        pytest.param(
+            ["--index"],
```

Comment:
I think we can clean these out now, done in #1130.

Reply:
PR merged so this is done now.